megablocks/layers/dmoe_test.py

```diff
@@ -137,7 +161,8 @@ def testdMoE_ForwardVersusMoE(
         out, _ = dmoe_mlp(x)
         self.assertSequenceEqual(out.shape, x.shape)
         self.assertSequenceEqual(expected_out.shape, x.shape)
-        self.assertTrue(testing.allclose(out, expected_out))
+        if num_remat_bits == -1:
```

Comment:
I'm assuming this was failing when using quantization? Can we loosen the tolerance for quantized tests, rather than disable completely?